examples/app-router/src/components/TestPage.tsx

```diff
@@ -4,19 +4,26 @@ import { usePathname } from 'next/navigation'
 
 type TestPageProps = {
   title: string
+  date: string
   buildTime: number
   expireTime?: number
 }
 
-export const TestPage: FC<TestPageProps> = ({ title, buildTime, expireTime }) => {
+export const TestPage: FC<TestPageProps> = ({ title, date, buildTime, expireTime }) => {
   const pathname = usePathname()
 
-  const revalidatePathHandler = () => fetch(`/api/revalidate?path=${pathname}`)
-
   const revalidateTagHandler = () => {
     const input = document.getElementById('tag') as HTMLInputElement | null
     if (input?.value) fetch(`/api/revalidate?tag=${input.value}`)
   }
+  const revalidatePathHandler = () => {
+    const input = document.getElementById('path') as HTMLInputElement | null
+    if (input?.value) {
```

Comment:
What do you expect to receive from `document.getElementById('path')`?

Reply:
if we want to get value from uncontrolled input let's use ref otherwise let's make it controlled

Reply:
In this case `document.getElementById('path')` I expect HTMLInputElement or null

Instead of `document.getElementById` I added ref and now it looks better 

Thanks